<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>logistic</class>
 <widget class="QDialog" name="logistic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Logistic</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>751</width>
     <height>321</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>741</width>
       <height>301</height>
      </rect>
     </property>
     <property name="title">
      <string>Studio</string>
     </property>
     <widget class="QTabWidget" name="tabWidget_logistic">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>701</width>
        <height>251</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Ajouter</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>671</width>
          <height>201</height>
         </rect>
        </property>
        <property name="title">
         <string>Ajouter un Studio</string>
        </property>
        <widget class="QLabel" name="label_18">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>30</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>ID_STUDIO</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_19">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>80</y>
           <width>51</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>NOM</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_20">
         <property name="geometry">
          <rect>
           <x>340</x>
           <y>70</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>DESCRIPTION</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_21">
         <property name="geometry">
          <rect>
           <x>350</x>
           <y>30</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>NUMERO</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_IDS">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>30</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_nomst">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>80</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_ST_numerost">
         <property name="geometry">
          <rect>
           <x>440</x>
           <y>30</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_descriptionst">
         <property name="geometry">
          <rect>
           <x>440</x>
           <y>70</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="ajout_studio">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>140</y>
           <width>131</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Valider</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Afficher</string>
       </attribute>
       <widget class="QTableView" name="tableView_studio">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>531</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_26">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>0</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Trier par</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_triStudio">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>0</y>
          <width>121</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_29">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>80</y>
          <width>71</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../Load.png</pixmap>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Modifier</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>681</width>
          <height>201</height>
         </rect>
        </property>
        <property name="title">
         <string>Modifier studio</string>
        </property>
        <widget class="QLabel" name="label_22">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>20</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>ID_STUDIO</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_23">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>60</y>
           <width>51</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>NOM</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_24">
         <property name="geometry">
          <rect>
           <x>370</x>
           <y>60</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>DESCRIPTION</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_25">
         <property name="geometry">
          <rect>
           <x>380</x>
           <y>20</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>NUMERO</string>
         </property>
        </widget>
        <widget class="QPushButton" name="modifier_studio">
         <property name="geometry">
          <rect>
           <x>234</x>
           <y>140</y>
           <width>131</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Modifier</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_IDS_2">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>20</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_nomst_2">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>60</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_ST_numerost_2">
         <property name="geometry">
          <rect>
           <x>480</x>
           <y>10</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_descriptionst_2">
         <property name="geometry">
          <rect>
           <x>480</x>
           <y>60</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Supprimer </string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>651</width>
          <height>201</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QPushButton" name="supprimer_studio">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>130</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Supprimer</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>90</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Identifiant du Studio</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_28">
         <property name="geometry">
          <rect>
           <x>450</x>
           <y>40</y>
           <width>101</width>
           <height>91</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../delete1.png</pixmap>
         </property>
        </widget>
        <widget class="QComboBox" name="supprimer_studio_2">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>90</y>
           <width>151</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>741</width>
       <height>311</height>
      </rect>
     </property>
     <property name="title">
      <string>PROGRAMME</string>
     </property>
     <widget class="QTabWidget" name="tabWidget_programme">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>721</width>
        <height>271</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Ajouter</string>
       </attribute>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>34</x>
          <y>90</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TYPE</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>24</x>
          <y>40</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>ID_PROGRAMME</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>34</x>
          <y>190</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>DATE</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>34</x>
          <y>140</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>NOM</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>40</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>ID_STUDIO</string>
        </property>
       </widget>
       <widget class="QPushButton" name="ajouter_prog">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>190</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Valider</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_idprog">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>40</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_typeprog">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>90</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_nomprog">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>140</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_dateprog">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>190</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_idst">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>40</y>
          <width>121</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Afficher</string>
       </attribute>
       <widget class="QTableView" name="tableView_prog">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>531</width>
          <height>211</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_31">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>70</y>
          <width>71</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../Load.png</pixmap>
        </property>
       </widget>
       <widget class="QPushButton" name="PDF">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>192</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>PDF</string>
        </property>
       </widget>
       <widget class="QPushButton" name="afficher_prog">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>160</y>
          <width>121</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>afficher</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Modifier</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>671</width>
          <height>231</height>
         </rect>
        </property>
        <property name="title">
         <string>Modifier un Programme</string>
        </property>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>54</x>
           <y>80</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>TYPE</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_14">
         <property name="geometry">
          <rect>
           <x>44</x>
           <y>30</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>ID_PROGRAMME</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_15">
         <property name="geometry">
          <rect>
           <x>54</x>
           <y>200</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>DATE</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_16">
         <property name="geometry">
          <rect>
           <x>54</x>
           <y>140</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>NOM</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_17">
         <property name="geometry">
          <rect>
           <x>320</x>
           <y>30</y>
           <width>61</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>ID_STUDIO</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="id_prog_edit">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>30</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLineEdit" name="type_prog_edit">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>80</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="nom_prog_edit">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>140</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="date_prog_edit">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>200</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="id_studiop_edit">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>30</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="modifier_prog">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>170</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Valider</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>Supprimer </string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>651</width>
          <height>211</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>80</y>
           <width>131</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Identifiant du Programme</string>
         </property>
        </widget>
        <widget class="QPushButton" name="supprimer_prog">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>130</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Supprimer</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_27">
         <property name="geometry">
          <rect>
           <x>460</x>
           <y>40</y>
           <width>101</width>
           <height>91</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../delete1.png</pixmap>
         </property>
        </widget>
        <widget class="QComboBox" name="supprimer_programme">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>80</y>
           <width>101</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="studio">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>340</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>STUDIO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="programme">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>340</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>PROGRAMME</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
